### DISCUSSION TITLE
Bones animations not exported

### DISCUSSION THREAD

**Discussion by ThomasGorisse** [sceneview/sceneform-android#111] [Q&A]
Hi,

Can someone help me exporting to gltf my bones animations form this:
[HandMotionMove-Animated-1.0.2.blend.zip](https://github.com/ThomasGorisse/sceneform-android-sdk/files/6752946/HandMotionMove-Animated-1.0.2.blend.zip)

Can't figure out what I'm doing wrong but even if the animation "ArmatureAction" appears in the glTF animations list, nothing is played.

Thanks

**Comment by grassydragon** [comment]
Hi!
I parented the mesh to the bone instead of using the Armature modifier.
[HandMotionMove-Animated-Updated.zip](https://github.com/ThomasGorisse/sceneform-android-sdk/files/7228323/HandMotionMove-Animated-Updated.zip)
